### PR TITLE
try to integrate new prototype into current website repo

### PR DESCRIPTION
**Pull request by SkrPaoWang**
Hi, I am trying to integrate new prototype into current open-cluster-management website repo. Currently I finish the homepage prototype. The next step is to finish the prototype for the other pages and makes the website looks better. Any contributions and comments are appreciated!

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@ baseURL = "http://open-cluster-management.io"
 languageCode = "en-us"
 title = "Open Cluster Management"
 
-theme = "hugo-geekdoc"
+theme = "exampleTheme"
```
Comment (qiujian16) [inline config.toml:5]: call it ocmTheme :)

File: themes/exampleTheme/layouts/_default/single.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                    <h1>{{ partial "title" . }}</h1>
+                    {{ partial "content" . }}
+                </article>
+            </div>
+        </main>
+    </div>
+{{ end }}
+
+
+
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/_default/single.html:23]: eof

File: themes/exampleTheme/layouts/index.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{{ define "main" }}
+{{- partial "block1.html" . -}}
+{{- partial "block2.html" . -}}
+{{- partial "block3.html" . -}}
+{{- partial "block4.html" . -}}
+{{- partial "block5.html" . -}}
+{{ end }}
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/index.html:7]: eof

File: themes/exampleTheme/layouts/partials/block1.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+>
+  <div class="container">
+    <div class="d-sm-flex align-items-center justify-content-between">
+      <div>
+        <h1><span class="text-warning"> Deliver apps accross
+          clusters and clouds
+        </span> with consistent configuration management</h1>
+        <p class="lead my-4">
+          A community-driven project focused on multicluster and
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block1.html:16]: change to

Make working with many Kubernetes clusters super easy regardless of where they are deployed.

File: themes/exampleTheme/layouts/partials/block1.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      <img
+      class="img-fluid w-20 d-none d-sm-block rounded pt-5 pe-1" 
+      
+      style="  width:30%; height:30%"
+      src="/robot7.png"
+      alt=""
+    />
+    </div>
+  </div>
+</section>
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block1.html:46]: eof

File: themes/exampleTheme/layouts/partials/block2.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+          <h2 style="text-align: justify;">Quick Start </h2>
+          </div>
+        <div class="col-5">
+          <div class="card  text-light"
+          
+          style="background-color: rgb(50, 0, 100);  padding:10px 50px 5px 10px">
+            <div class="card-body text-center" >
+              <h2 style="text-align: justify;  color: #00FFFF;">1</h2>
+              <h6 style="text-align: justify;  color: white; font-size: 18.5px;">
+              Install a Cluster Manager on your hub cluster</h6>
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block2.html:17]: I would simply go with 

To bootstrap the open-cluster-management control plane, run
```
clusteradm init 
```
And remove part of Install Cluster Manager and Install Klusterlet

File: themes/exampleTheme/layouts/partials/block3.html [hunk 7]
```diff
@@ -0,0 +1,19 @@
+<!-- main block 3 -->
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block3.html:1]: let's remove this section

File: themes/exampleTheme/layouts/partials/block5.html [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+                <br>
+                <div class="p-1" style="text-align:left">
+                <button type="button" class="btn btn-outline-info p-2 ps-2 pe-2">
+                  <div class="text-center  fs-6 ">Learn more about applications</div></button>
+              </div></div>
+      
+            </div>
+          </div>
+
+          <div class="col-4">
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block5.html:49]: this part is not included, we need to remove it. We could add a part saying 

And more addons...

File: themes/exampleTheme/layouts/partials/block5.html [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+            <div class="card text-light" style="background-color: rgb(50, 0, 100);">
+
+              <div class="card-body text-center">
+                <p class=" mb-3  fs-4 pe-3" style="text-align:left;
+                ">
+                Configure, secure, and manage your resources.</p>
+         
+                <p class="card-text fw-light" style="text-align:left;
+                font-size: 14px;">
+                  Policy and configuration manageement uses labels to
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block5.html:16]: manageement --> management

File: themes/exampleTheme/layouts/partials/block5.html [hunk 10]
```diff
@@ -0,0 +1,72 @@
+<section id="instructors" class="p-5 "
```
Comment (qiujian16) [inline themes/exampleTheme/layouts/partials/block5.html:1]: I would put a title for this section is open-cluster-management addons

**Review by mikeshng — COMMENTED** [review]
This looks amazing! Thank you for your work. Here are some of my comments:
- **Please ensure the website looks OK on mobile devices as well. Most importantly the front/home page** 
- The contents from the prototype are outdated. I think it's better to use some of the wording from https://open-cluster-management.io
- If we can have the favicon to be the new logo that will be good
- The page current on the windows tab says `Frontend Bootcamp`
- The top/bottom right icons (twitter, youtube, github, info) are not working. 
- Other links on the homepage doesn't seem to work.
- Missing the `© Open Cluster Management Authors 2021`

**Pull request closed without merging**